### PR TITLE
Fix #1338 Project creation outline panel should use the project dashboard styles

### LINKED ISSUE #1338: [$60] Project creation outline panel should use the project dashboard styles
Project creation outline panel isn't following the appropriate styles. We should reuse the project dashboard header styles.

Current:
screen shot 8

Expected:
screen shot 9

### PR DESCRIPTION
**Pull request by zjuasmn**

### CODE REVIEW

File: src/projects/create/components/ProjectOutline.js [hunk 1]
```diff
@@ -21,7 +22,7 @@ function ProjectOutline({ project }) {
   return (
     <div className="ProjectOutline">
       <h5>Project Outline</h5>
-      <div className="project-name">{ project.name }</div>
+      <ProjectCardHeader project={project} />
       <div className="project-product">{ productName }</div>
```
Comment (vikasrohit) [inline src/projects/create/components/ProjectOutline.js]: Do we need this block now?

**Comment by gondzo** [comment]
@zjuasmn The styles don't really match. See bellow
Project outline
image
Creating a new project
image

**Comment by zjuasmn** [comment]
@gondzo The image below "Creating a new project" is not from my PR.

File: src/projects/create/components/ProjectOutline.js (hunk 1 above)
Comment (zjuasmn) [reply src/projects/create/components/ProjectOutline.js]: Do you mean I should remove the header?

File: src/projects/create/components/ProjectOutline.scss [hunk 2]
```diff
@@ -33,11 +24,8 @@
   }
 
   .project-description {
-    line-height: 5*$base-unit;
-    margin-top: $base-unit * 4;
```
Comment (zjuasmn) [inline src/projects/create/components/ProjectOutline.scss]: @gondzo I did remove the margin-top of project-description

File: src/projects/create/components/ProjectOutline.js (hunk 1 above)
Comment (gondzo) [reply src/projects/create/components/ProjectOutline.js]: I guess we do need it, there was no request to remove project name from the outline panel. I'll merge this but please raise the issue if I'm missing something. @vikasrohit

Comment (vikasrohit) [reply src/projects/create/components/ProjectOutline.js]: @gondzo I was thinking about `productName`. I would check once all changes on in dev.

**Pull request merged**